src/main/java/se/sundsvall/supportmanagement/api/EventResource.java

```diff
@@ -0,0 +1,63 @@
+package se.sundsvall.supportmanagement.api;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static org.springframework.http.MediaType.APPLICATION_PROBLEM_JSON_VALUE;
+import static org.springframework.http.ResponseEntity.ok;
+
+import java.util.Collections;
+
+import org.springdoc.api.annotations.ParameterObject;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+import org.zalando.problem.Problem;
+import org.zalando.problem.violations.ConstraintViolationProblem;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import se.sundsvall.dept44.common.validators.annotation.ValidUuid;
+import se.sundsvall.supportmanagement.api.model.event.Event;
+import se.sundsvall.supportmanagement.service.EventService;
+
+@RestController
+@Validated
+@Tag(name = "Errand events", description = "Errand event operations")
+public class EventResource {
+
+	@Autowired
+	private EventService eventService;
+
+	/**
+	 * Class needed to solve swagger implementation schema
+	 */
+	static class PagedEvent extends PageImpl<Event> {
+		private static final long serialVersionUID = 1L;
+
+		public PagedEvent() {
+			super(Collections.emptyList());
+		}
+	}
+
+	@GetMapping(path = "/errands/{id}/events", produces = { APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE })
+	@Operation(summary = "Read errand events", description = "Returns all existing events for the errand that matches the provided id")
+	@ApiResponse(responseCode = "200", description = "Successful operation", content = @Content(mediaType = APPLICATION_JSON_VALUE, schema = @Schema(implementation = PagedEvent.class)))
```

Comment:
Why not:
`@ApiResponse(responseCode = "200", description = "Successful operation", content = @Content(mediaType = APPLICATION_JSON_VALUE), useReturnTypeSchema = true)`

Then remove: PagedEvent 

Have not tested, but will probably wortk.

Reply:
I tried that but it didn't work - which was the reason that I added the PagedEvent class. 

With suggested solution:
![image](https://github.com/Sundsvallskommun/api-service-support-management/assets/56998189/ac706316-be85-4523-a00d-b1ed5725dff2)

With PagedEvent:
![image](https://github.com/Sundsvallskommun/api-service-support-management/assets/56998189/e4fc417a-8308-4d98-9db2-5befe701171e)

Reply:
It works if you do it like this:
`@ApiResponse(responseCode = "200", description = "Successful operation", useReturnTypeSchema = true)`

Reply:
It seems to work if you do it like this:
`@ApiResponse(responseCode = "200", description = "Successful operation", useReturnTypeSchema = true)`

Reply:
Good finding 👍, I'll refactor to use returnTypeSchema instead.